src/Opdex.Platform.WebApi/Controllers/AuthController.cs

```diff
@@ -130,7 +130,7 @@ public ActionResult<string> GetStratisId()
             SigningCredentials = new SigningCredentials(key, SecurityAlgorithms.HmacSha256Signature)
         };
 
-        tokenDescriptor.Subject.AddClaim(new Claim("wallet", body.PublicKey));
+        tokenDescriptor.Subject.AddClaim(new Claim(JwtRegisteredClaimNames.Sub, body.PublicKey));
```

Comment:
The highlighted change is simply retrieving the users wallet address from the "sub" claim rather than the "wallet" claim. If you look below in this file, the admin key is being retrieved from the http request header. I've kept in the requirement to be logged in with a wallet address that exists in the admin table, as an additional check.

Reply:
I see now, my bad